web/docs/operators/context.md

```diff
@@ -59,6 +59,10 @@ See the [list of available context types](../contexts.md).
 
 Context-specific options in the format `--key value` or `--flag`.
 
+### `--version <version>`
+
+Context-specific version number for the binary data to load.
+
```

Comment:
I know that this was straightforward to implement, but from a user-perspective, what's the use case for it? Does the user ever need to think about versioning?

Reply:
You might be right, the real use cases for this feature are very slim, and it probably isn't worth the increased API surface. It's not every day one has to `context load` something that was `context save`d with a different context binary version.

Reply:
I'd be fine with leaving it in as well, but we should probably have a note that explains what it's for in the docs then.

Reply:
Eh, I'll rip it out. It's easy to add if there's customer or developer demand. The version isn't user-queryable anyway, so it's difficult for a user to find out what exact value to put there.